src/gobupload/relate/update.py

```diff
@@ -255,13 +252,19 @@ def __init__(self, src_catalog_name, src_collection_name, src_field_name):
         self.src_collection_name = src_collection_name
         self.src_field_name = src_field_name
 
-        self.src_collection = self.model.get_collection(src_catalog_name, src_collection_name)
+        try:
+            self.src_collection = self.model[src_catalog_name]['collections'][src_collection_name]
+        except KeyError:
+            self.src_collection = None
```

Comment:
I don't think this is a better solution than the helper function in gob_model. Now we're repeating this try/except pattern everywhere, which is harder to read than a method call.

Reply:
Removed unnecessary try/except KeyError.